From 5681d9bcef0bf76453b387cad325cd6c3c527919 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <lass@mail.uni-paderborn.de>
Date: Tue, 12 Sep 2017 19:33:15 +0200
Subject: [PATCH 4/6] vol: add missing include of stdint.h to volinodes.h

volinodes.h requires stdint.h for use of uintptr_t:

In file included from vutil.c:55:0:
volinodes.h: In function 'init_inode_info':
volinodes.h:109:45: error: 'uintptr_t' undeclared (first use in this function); did you mean 'intptr_t'?
  stuff[i].inode = (Inode*)((char*)header + (uintptr_t)stuff[i].inode);
                                             ^~~~~~~~~
                                             intptr_t

Similar changes have been applied earlier in 1e53da3.

This change does not apply to master since e2b4fde added an include of roken.h
to all affected source files which in turn includes stdint.h.

Change-Id: Iddd174e844583e06f66d910052e11feff928ba40
---
 src/vol/volinodes.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/vol/volinodes.h b/src/vol/volinodes.h
index 181b441d9..2dec0ceeb 100644
--- a/src/vol/volinodes.h
+++ b/src/vol/volinodes.h
@@ -18,6 +18,9 @@
 #define __volinodes_h_
 
 #include <stddef.h>
+#ifdef HAVE_STDINT_H
+# include <stdint.h>
+#endif
 
 /* Used by vutil.c and salvager.c */
 
-- 
2.15.1

